compatibility/common/src/test/java/org/projectnessie/tools/compatibility/internal/TestNessieCompatibilityExtensions.java

```diff
@@ -400,20 +419,32 @@ void never() {
 
   @ExtendWith(OlderNessieClientsExtension.class)
   @ExtendWith(OlderNessieServersExtension.class)
+  @SuppressWarnings({
+    "JUnitMalformedDeclaration",
+    "NewClassNamingConvention"
+  }) // Intentionally not nested, used above
   static class TooManyExtensions1 {
     @Test
     void testSome() {}
   }
 
   @ExtendWith(OlderNessieClientsExtension.class)
   @ExtendWith(NessieUpgradesExtension.class)
+  @SuppressWarnings({
+    "JUnitMalformedDeclaration",
+    "NewClassNamingConvention"
+  }) // Intentionally not nested, used above
   static class TooManyExtensions2 {
     @Test
     void testSome() {}
   }
 
   @ExtendWith(NessieUpgradesExtension.class)
   @ExtendWith(OlderNessieServersExtension.class)
+  @SuppressWarnings({
+    "JUnitMalformedDeclaration",
+    "NewClassNamingConvention"
+  }) // Intentionally not nested, used above
```

Comment:
IntelliJ auto-collapses this into a single line :)
IMO this way makes it more likely that auto-formatting will keep the comment attached to the annotation
Leaving as-is since nit, although I also do not feel strongly